tests/test_misc.py

```diff
@@ -2302,7 +2302,7 @@ def test_emergencyrecover(node_factory, bitcoind):
     """
     Test emergencyrecover
     """
-    l1, l2 = node_factory.get_nodes(2, opts=[{}, {}])
+    l1, l2 = node_factory.get_nodes(2, opts=[{'allow_broken_log': True}, {}])
```

Comment:
`allow_broken_log: True`
Because [https://github.com/ElementsProject/lightning/blob/ad3cbed7c28b2d4e9327885f6c3763f8f4acef41/onchaind/onchaind.c#L3806]